editor/block-toolbar/index.js

```diff
@@ -20,18 +21,53 @@ import BlockSwitcher from '../block-switcher';
 import BlockMover from '../block-mover';
 import BlockRightMenu from '../block-settings-menu';
 
+/**
+ * Module Constants
+ */
+const { ESCAPE, F10 } = keycodes;
+
 function FirstChild( { children } ) {
 	const childrenArray = Children.toArray( children );
 	return childrenArray[ 0 ] || null;
 }
 
+function isMac() {
```

Comment:
It looks like it could be moved to the file with browser's utils or should we wait until it's used somewhere else?